### PR TITLE
echoing a little less

### PR DESCRIPTION
**Pull request by cjekal**
Users have complained that the echo was too noisy/chatty. PO requested we reduce the chattiness to keep users happy.

### CODE REVIEW

File: src/main/java/com/example/demo/controllers/EchoController.java [hunk 1]
```diff
@@ -12,6 +12,6 @@ public class EchoController {
     @PostMapping("/echo")
     public String echo(@RequestBody String value) {
         System.out.println("haha!");
-        return value + " ... " + value + " main branch! one more change!";
+        return value + " ... " + value + " a little less!";
```
Comment (cjekal) [inline src/main/java/com/example/demo/controllers/EchoController.java:15]: this still seems quite chatty! should we reduce even more?

**Review by cjekal — COMMENTED** [review]
leaving a comment for you, charles

File: src/main/java/com/example/demo/controllers/EchoController.java (hunk 1 above)
Comment (KatyaMa) [reply src/main/java/com/example/demo/controllers/EchoController.java:15]: looks good. Katya

Comment (ntr6) [reply src/main/java/com/example/demo/controllers/EchoController.java:15]: Unlike the users, I appreciate chattiness.

Comment (Arvind0729) [reply src/main/java/com/example/demo/controllers/EchoController.java:15]: It looks great

**Review by ntr6 — APPROVED** [review]

**Review by KatyaMa — APPROVED** [review]

**Review by ntr6 — CHANGES_REQUESTED** [review]
I am requesting changes, because I don't want to reduce chattiness.

**Pull request merged**